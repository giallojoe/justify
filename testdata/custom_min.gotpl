# tiny custom template to prove overrides flow through
build:
	@echo CMAKE={{index .CMakeDirs 0}}
program:
	@echo CARGO={{.CargoBinGuess}}
node:
	@echo NODES={{range .NodeEntries}}{{.}}|{{end}}
attach:
	@echo ATTACH={{.AttachOnDev}}
