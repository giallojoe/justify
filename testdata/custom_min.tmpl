# tiny custom template to prove overrides flow through
build:
	@echo CMAKE={{index .CMakeDirs 0}}
program:
	@echo CARGO={{.CargoBinGuess}}
node:
	@echo NODES={{range .NodeEntries}}{{.}}|{{end}}
go:
	@echo GOEXE={{range .GoExeCandidates}}{{.}}|{{end}}
attach:
	@echo ATTACH={{.AttachOnDev}}
