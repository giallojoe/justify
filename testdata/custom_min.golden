# tiny custom template to prove overrides flow through
build:
	@echo CMAKE=cmake-out
program:
	@echo CARGO=crate_name
node:
	@echo NODES=dist/server/entry.mjs|
attach:
	@echo ATTACH=false
